.nuke/build.schema.json

```diff
@@ -6,6 +6,15 @@
     "build": {
       "type": "object",
       "properties": {
+        "Configuration": {
+          "type": "string",
+          "description": "The solution configuration to build. Default is 'Debug' (local) or 'CI' (server)",
+          "enum": [
+            "CI",
+            "Debug",
+            "Release"
```

Comment:
There is no corresponding "Configuration.Release" 🤔

Reply:
Fixed that. Forgot to include the updated `json